staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_aggregator.go

```diff
@@ -110,6 +144,11 @@ type byPriority struct {
 func (a byPriority) Len() int      { return len(a.specs) }
 func (a byPriority) Swap(i, j int) { a.specs[i], a.specs[j] = a.specs[j], a.specs[i] }
 func (a byPriority) Less(i, j int) bool {
+	// All local specs will come first
+	// WARNING: This will result in not following priorities for local APIServices.
+	if a.specs[i].apiService.Spec.Service == nil {
```

Comment:
I've commented on this PR about this issue. I think order should not matter for local delegates at all. I sent this PR (https://github.com/kubernetes/kube-openapi/pull/4) to make it possible to Merge specs without renaming definitions. All local delegates should use newly introduced SafeMergeSpec and if we do so, we don't need to worry about order (I would even move them out of this sorting if we agree on this).